### DISCUSSION TITLE
[FR] Movable Float Button

### DISCUSSION THREAD

**Discussion by Moyf** [chrisgurney/obsidian-note-toolbar#122] [Ideas]
Hi!

Some extension button could be directly drag&move, I wonder could it be also implemented for NTB float button?
![d95e4865-4b2f-4c88-987e-0f8a35996666](https://github.com/user-attachments/assets/fc593d82-2cb1-4742-88b2-f59759be73d3)

![image](https://github.com/user-attachments/assets/ab0e4e77-8045-480c-ba0a-5d67dfa78ed5)
↑ Drag & Move

**Comment by chrisgurney** [comment]
Thanks @Moyf ! [Somebody else](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/78) noted this was doable in the [vConsole plugin](https://github.com/zhouhua/obsidian-vconsole) but I haven't pinpointed the code that allows for this. 

I think I'd definitely like to support something like this, versus having to position it in CSS / Style Settings.

**Reply by Moyf** [reply]
Found a refrence: [How to drag an element using javascript](https://devdojo.com/tnylea/how-to-drag-an-element-using-javascript)

**Reply by chrisgurney** [reply]
@Moyf Thanks, that's great!
